index.php

```diff
@@ -23,17 +51,7 @@
     There are <a href="<?php print $portdb_url; ?>">
     <b>thousands</b> of ports</a> in our tree, distributed among different categories, and more are being added on a regular basis.</p>
 
-    <h3 class="subhdr">Getting started</h3>
-
-    <p>For information on installing MacPorts please see the <a href="install.php">installation</a> section of this site and
-    explore the myriad of download options we provide and our base system requirements.</p>
-
-    <p>If you run into any problems installing and/or using MacPorts we also have many options to help you, depending on how
-    you wish to get <a href="contact.php">get in touch with us</a>. Other important help resources are our online documentation,
-    A.K.A <a href="<?php print $guide_url; ?>"> The MacPorts Guide</a>, and our Trac <a href="<?php print $trac_url; ?>">Wiki
-    server &amp; bug tracker</a>.</p>
 
-    <p><b>Latest MacPorts <a href="install.php">release</a>: <?php print $macports_version_latest; ?></b></p>
```

Comment:
Maybe we can keep the version number in the download link above? I think it makes sense to state that somewhere.